tests/main_test.py

```diff
@@ -21,8 +21,33 @@
         'mock_func': ['getenv'],
         'mock_ret': ['12345678;23456789'],
         'is_async': False
+    },
+    {
+        'func': main.check_ok,
+        'expected_res': [True],
+        'arg': ('http://example.org/',),
+        'is_async': False
+    },
+    {
+        'func': main.check_ok,
+        'expected_res': [False],
+        'arg': ('garbage',),
+        'is_async': False
+    },
+    {
+        'func': main.check_ping,
+        'expected_res': [True],
+        'arg': ('example.org',),
+        'is_async': False
+    },
+    {
+        'func': main.check_ping,
+        'expected_res': [False],
+        'arg': ('garbage',),
+        'is_async': False
     }
 ]
+#last 4 tests are incomplete; might want to mock some calls
```

Comment:
- [ ] Add test to a page that doesn't exist in order to obtain a 404
- [ ] Is really helpful test without a valid URL?
- [ ] `check_ping` should mock `call` in order to obtain some useful results

Reply:
1) done
2) not really, the urls list is managed by us
3) not needed